xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp

```diff
@@ -2450,6 +2486,9 @@ void CMixer::InitCycle()
 
   m_processPicture.DVDPic = m_mixerInput[1].DVDPic;
   m_processPicture.videoSurface = m_mixerInput[1].videoSurface;
+
+  // update deinterlacing method used in processInfo (none if progressive)
+  m_config.processInfo->SetVideoDeintMethod(deintStr);
```

Comment:
you still call m_config.processInfo->SetVideoDeintMethod in InitCycle, which is wrong